src/shared/sol-random.c

```diff
@@ -91,7 +91,11 @@ get_platform_seed(uint64_t seed)
 
     /* Fall back to using a bad source of entropy if platform-specific,
      * higher quality random sources, are unavailable. */
+#ifdef SOL_PLATFORM_ZEPHYR
+    return ((uint64_t)sys_rand32_get() << 32) | sys_rand32_get();
```

Comment:
It *used* to be on always, recently it seems we need a config (and the config was changed wrongly, I had to patch the Kconfig) :/ But it's the recommended way of getting an almost-random number in the OS, it seems.